environments/prod/variables.tf

```
@@ -0,0 +1,20 @@
+variable "allowed_account_ids" {
+  description = "List of allowed AWS account ids where resources can be created"
+  type        = list(string)
+  default     = []
+}
```

Probably we will need just one account per environment.
```suggestion
variable "allowed_account_id" {
  description = "Allowed AWS account id where resources can be created"
  type        = string
}
```

---

True!